stdlib/Distributed/test/distributed_exec.jl

```diff
@@ -132,6 +132,18 @@ end
 testf(id_me)
 testf(id_other)
 
+function loop_until_false(f::Function; timeout_seconds::Integer = 60)
+    start_time = time_ns()
+    while f() != false
+        sleep(1)
+        if ( ( time_ns() - start_time )/1e9 ) > timeout_seconds
+            @error "Timed out" timeout_seconds
+            return nothing
+        end
+    end
+    return nothing
+end
```

Comment:
I think extracting this out as a function is a good idea. But I think it'd be also nice to easily make sure that the condition we use for polling and the condition we use for testing are the same. So, how about returning `true` on success and `false` on timeout?

Also, I think a name like `poll_while` is more informative (i.e., we are not using a condition variable or something).

Reply:
Both of those sound good to me! If you make them as "suggestions", I can accept them.

Reply:
```suggestion
function poll_while(f::Function; timeout_seconds::Integer = 60)
    start_time = time_ns()
    while f() != false
        sleep(1)
        if ( ( time_ns() - start_time )/1e9 ) > timeout_seconds
            @error "Timed out" timeout_seconds
            return false
        end
    end
    return true
end
```